doc/user/install-operator-sdk.md

```diff
@@ -86,13 +86,17 @@ $ chmod +x operator-sdk-${RELEASE_VERSION}-x86_64-apple-darwin && sudo mkdir -p
 - [go][go_tool] version v1.13+.
 
 ```sh
-$ go get -d github.com/operator-framework/operator-sdk # This will download the git repository and not install it
-$ cd $GOPATH/src/github.com/operator-framework/operator-sdk
+$ git clone https://github.com/operator-framework/operator-sdk
+$ cd operator-sdk
 $ git checkout master
 $ make tidy
 $ make install
 ```
 
+Please note, on some Linux distributions you will need to set your `GOPROXY`
+environment variable. Ensure that it is set to the following,
+`GOPROXY="https://proxy.golang.org,direct"`.
```

Comment:
I don't think this is a distribution-specific requirement. Why do you think it is out of curiosity?

Reply:
based on my experience with the default go installation in fedora, and these two comments on the issue i raised:

https://github.com/operator-framework/operator-sdk/issues/2573#issuecomment-587089134
> In the GO 1.13+, the GOPROXY is by default `https://proxy.golang.org,direct` so because of this has no need to add in the docs. For someone reason, your go env was not set up properly.

https://github.com/operator-framework/operator-sdk/issues/2573#issuecomment-587091615
> on Fedora 31, you are required to have GOPROXY set, this is due to the way golang is packaged on Fedora, I found this out when doing the 1.15.0 release on my Fedora server.

it leads me to believe that on some distributions (eg the one that @camilamacedo86 is using), the `GOPROXY` is set by default to the location referenced in this pr. it certainly does not seem to be set that way on fedora 31, curiously enough i think this might work on earlier versions of fedora.

i can make that statement more generic though, just advising to check your `GOPROXY`, seems like sound advice regardless. i spent several hours scratching my head and looking for the root cause on issue #2574 due to fedora's configuration.

Reply:
WDYT about ?

```suggestion
**Note:**  Ensure that your `GOPROXY` is set with its default values for go versions 1.13+ which is `https://proxy.golang.org,direct`
```

@elmiko, 

Note that I know that the GOPROXY has these values by default for go 1.13+ and I do NOT know why you did not have it in your env but also I am afraid that it has NO relation with the distro.

Reply:
++, i'm fine with that change.

my only mention of the distro is that it looks like fedora explicitly _does not_ set GOPROXY. my thought was that some distros (perhaps the one you use) do set it. regardless, i agree that the docs should be more generic in nature with regards to this issue.

Reply:
This might be for certain package managers or packaging repositories, not distro-specific. Regardless I agree with this change.